Mathlib/GroupTheory/GroupAction/Quotient.lean

```diff
@@ -404,6 +404,7 @@ noncomputable def equivSubgroupOrbitsQuotientGroup [IsPretransitive α β]
     convert one_mem H
     rw [inv_mul_eq_one, eq_comm, ← inv_mul_eq_one, ← Subgroup.mem_bot, ← free (g⁻¹ • x),
         mem_stabilizer_iff, mul_smul, (exists_smul_eq α (g⁻¹ • x) x).choose_spec]
+    infer_instance
```

Comment:
Should this one also come with an adaptation note?
```suggestion
  #adaptation_note
  /--
  After https://github.com/leanprover/lean4/pull/5376 we need to search for this instance explicitly.
  -/
    infer_instance
```

Reply:
More specifically, it's that `convert` isn't solving it by typeclass search anymore, but just spitting out an unsolved implicit argument.

If there's anything to do here, it is to change the behaviour of `convert` to more agressively solve such goals with `infer_instance` itself.

Reply:
```suggestion
    #adaptation_note
    /--
    After https://github.com/leanprover/lean4/pull/5376 we need to search for this instance explicitly.
    TODO: change the behaviour of `convert` to more agressively solve such goals with `infer_instance` itself.
    -/
    infer_instance
```